### PR TITLE
Correct Read Error Format

### PR DESCRIPTION
**Pull request by EasterTheBunny**
Changed error text to be `read error` instead of `rpc error` to limit confusion. Batch flag to string corrected to show `true|false` instead of bool type.

### CODE REVIEW

File: core/services/relay/evm/read/errors.go [hunk 1]
```diff
@@ -43,8 +43,8 @@ func newErrorFromCall(err error, call Call, block string, batch bool) ErrRead {
 func (e ErrRead) Error() string {
 	var builder strings.Builder
 
-	builder.WriteString("[rpc error]")
-	builder.WriteString(fmt.Sprintf(" batch: %T;", e.Batch))
+	builder.WriteString("[read error]")
+	builder.WriteString(fmt.Sprintf(" batch: %t;", e.Batch))
```
Comment (ilija42) [inline core/services/relay/evm/read/errors.go]: maybe if its false just omit the  batch word?

Comment (jmank88) [reply core/services/relay/evm/read/errors.go]:
```suggestion
	builder.WriteString("[read error]")
	fmt.Fprintf(&builder, " batch: %t;", e.Batch))
```

File: core/services/relay/evm/read/errors.go [hunk 2]
```diff
@@ -43,10 +43,13 @@ func newErrorFromCall(err error, call Call, block string, batch bool) ErrRead {
...
 
-	builder.WriteString("[rpc error]")
-	builder.WriteString(fmt.Sprintf(" batch: %T;", e.Batch))
+	builder.WriteString("[read error]")
 	builder.WriteString(fmt.Sprintf(" err: %s;", e.Err.Error()))
 
+	if e.Batch {
+		builder.WriteString(" batch;")
+	}
+
```
Comment (ilija42) [inline core/services/relay/evm/read/errors.go]: does it make sense to have an enum for each type of  CR read call?

Comment (EasterTheBunny) [reply core/services/relay/evm/read/errors.go]: I'm not sure what you mean. What information would the enum capture?

Comment (ilija42) [reply core/services/relay/evm/read/errors.go]: batch/queryKey/GetLatestValue

File: core/services/relay/evm/read/errors.go [hunk 3]
```diff
@@ -10,9 +10,17 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/logpoller"
 )
 
-type ErrRead struct {
+type readType string
+
+const (
+	batchReadType  readType = "BatchGetLatestValue"
+	singleReadType readType = "GetLatestValue"
```
Comment (ilija42) [inline core/services/relay/evm/read/errors.go:17]: really a nit, but why not latestReadType

Comment (EasterTheBunny) [reply core/services/relay/evm/read/errors.go:17]: Because I was thinking on the line of batch vs not batch, thus batch and single. ¯\_(ツ)_/¯

**Review by ilija42 — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Review by Farber98 — APPROVED** [review]

**Pull request merged**